src/component/UserProfile/UserRegister.tsx

```diff
@@ -377,9 +453,32 @@ export function UserRegister(props: userRegisterProps) {
           helperText={passwordErrorTxt}
         />
 
+          <FormControl sx={{ width: '100%' }}>
+              <InputLabel id="community-multiple-checkbox-label">Watch Group</InputLabel>
+              <Select
+                  labelId="community-multiple-checkbox-label"
+                  id="community-multiple-checkbox"
+                  multiple
+                  value={selectedCommunities}
+                  onChange={handleCommunityChange}
+                  input={<OutlinedInput label="Tag" />}
+                  renderValue={(selected) => selected.join(', ')}
+                  MenuProps={CommunitySelectProps}
+                  error={!communityValid}
+              >
+                  {availCommunities.map((community) => (
+                      <MenuItem key={community.id} value={community.name}>
+                          <Checkbox checked={selectedCommunities.indexOf(community.name) > -1} />
+                          <ListItemText primary={community.name} />
+                      </MenuItem>
+                  ))}
+              </Select>
+            <FormHelperText sx={{ color: '#e74c3c' }} hidden={communityValid}>{communityErrorTxt}</FormHelperText>
+          </FormControl>
+
         <Box
           component='span'
-          sx={{ m: 1, color: 'red' }}
+          sx={{ m: 1, color: '#e74c3c' }}
           hidden={submitErrorTxt === ''}
```

Comment:
this logic isn't going to hit anymore with your changes since we're not currently setting submitErrorTxt